code_schemes/demographics/kenya_constituency.json

```diff
@@ -1995,13 +1995,13 @@
     },
     {
       "CodeID": "code-c24cb56f",
-      "DisplayText": "mbita",
+      "DisplayText": "suba north/mbita",
       "CodeType": "Normal",
       "NumericValue": 254,
-      "StringValue": "mbita",
+      "StringValue": "suba_north_mbita",
       "VisibleInCoda": true,
       "MatchValues": [
-        "mbita"
+        "suba north/mbita"
```

Comment:
Check which of these needs to match what it's called in CoreDataModules (e.g. stringvalues in mapping?)

Reply:
string value it is: https://github.com/AfricasVoices/CoreDataModules/blob/3e2d90a37717f802c5b5169992305c4d94b68131/core_data_modules/analysis/mapping/participation_maps.py#L59